app/assets/javascripts/annotations.js

```diff
@@ -644,7 +644,10 @@ function newAnnotationBoxForPDF(annObj) {
     $(body).hide();
     $(edit).hide();
     $(score).hide();
+    $(minimize).hide();
     var form = newEditAnnotationForm(annObj.line, annObj);
+    $(box).draggable( 'disable' );
+    $(box).resizable( 'disable' );
```

Comment:
Making the box resizing "disable" can cause users to expand the comment box and hide the buttons since the box is not expanding. I suggest either getting rid of this line or restricting the expansion of the comment box.

![Screen Shot 2019-12-05 at 1 06 33 AM](https://user-images.githubusercontent.com/31053044/70208444-932ca380-16fb-11ea-98d8-c8b27d081e16.png)

![Screen Shot 2019-12-05 at 1 09 20 AM](https://user-images.githubusercontent.com/31053044/70208564-e7378800-16fb-11ea-9643-1fca34449246.png)

Reply:
Fixed the edit form expandable.